build/windows/installer/Installer.sln

```diff
@@ -0,0 +1,40 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{B7DD6F7E-DEF8-4E67-B5B7-07EF123DB6F0}") = "Installer", "Installer.wixproj", "{340293B0-F46C-46A0-88D8-4BB2F3465C53}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
```

Comment:
I'm curious, what are these `Debug` builds for? Does the installer create an installer for all the platforms specified in this file?

Reply:
Visual Studio loves generating those. I might be able to remove some.